README.md

```diff
@@ -1,5 +1,5 @@
 # Awesome programming languages
-The list of 222 awesome programming languages that you might be interested in.
+The list of 223 awesome programming languages that you might be interested in.
```

Comment:
Counters have been already updated [here](https://github.com/ChessMax/awesome-programming-languages/commit/c1029c86b40ca7d12c0bb3bb7398bdb534afb516).

Reply:
hm? I don't understand, the number should be correct. do you need me to revert the commit?

Reply:
It seems that everything is OK. So no reverting is required.